Apps/AdminWebClient/src/ClientApp/src/views/Feedback.vue

```diff
@@ -151,6 +183,25 @@ export default class FeedbackView extends Vue {
         return this.adminTags.map<string>((x) => x.name);
     }
 
+    private get filterIsActive(): boolean {
+        return this.selectedAdminTagIds.length > 0;
+    }
+
+    private get filteredFeedbackList(): UserFeedback[] {
+        const includedInSelectedTags = (tag: AdminTag) =>
+            this.selectedAdminTagIds.includes(tag.id);
+
+        const anyTagsMatchFilter = (feedbackTags: UserFeedbackTag[]) =>
+            this.selectedAdminTagIds.length === 0 ||
+            feedbackTags.filter((feedbackTag) =>
+                includedInSelectedTags(feedbackTag.tag)
+            ).length > 0;
+
+        return this.feedbackList.filter((userFeedback) =>
```

Comment:
Looks like the only method that is doing actual work is the one above. I think a single method would be sufficient for the filtering

Reply:
Bedgrudingly agreed.